src/Aspire.Hosting.Azure.AppService/api/Aspire.Hosting.Azure.AppService.cs

```diff
@@ -0,0 +1,45 @@
+//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+namespace Aspire.Hosting
+{
+    public static partial class AzureAppServiceComputeResourceExtensions
+    {
+        public static ApplicationModel.IResourceBuilder<T> PublishAsAzureAppServiceWebsite<T>(this ApplicationModel.IResourceBuilder<T> builder, System.Action<Azure.AzureResourceInfrastructure, global::Azure.Provisioning.AppService.WebSite> configure)
+            where T : ApplicationModel.IComputeResource { throw null; }
+    }
+
+    public static partial class AzureAppServiceEnvironmentExtensions
+    {
+        public static ApplicationModel.IResourceBuilder<Azure.AppService.AzureAppServiceEnvironmentResource> AddAzureAppServiceEnvironment(this IDistributedApplicationBuilder builder, string name) { throw null; }
+    }
+}
+
+namespace Aspire.Hosting.Azure
+{
+    public sealed partial class AzureAppServiceWebsiteCustomizationAnnotation : ApplicationModel.IResourceAnnotation
+    {
+        public AzureAppServiceWebsiteCustomizationAnnotation(System.Action<AzureResourceInfrastructure, global::Azure.Provisioning.AppService.WebSite> configure) { }
+
+        public System.Action<AzureResourceInfrastructure, global::Azure.Provisioning.AppService.WebSite> Configure { get { throw null; } }
+    }
+}
+
+namespace Aspire.Hosting.Azure.AppService
```

Comment:
```suggestion
namespace Aspire.Hosting.Azure
```

? The ACA one is under `namespace Aspire.Hosting.Azure.AppContainers;`. But most of the Azure resource classes are just under `namespace Aspire.Hosting.Azure`.

Reply:
https://github.com/dotnet/aspire/pull/9285